.gitignore

```diff
@@ -2,3 +2,10 @@ vendor/
 cmd/commento/commento
 *.db
 .idea
+.env
```

Comment:
You can still use the `.env*`, then exclude the single one to allow with `!` e.g. `!.env.notignored`